src/app/components/shared/input/Input.tsx

```diff
@@ -0,0 +1,35 @@
+import clsx, { ClassValue } from 'clsx';
+import { InputHTMLAttributes } from 'react';
+
+interface Props
+  extends Pick<
+    InputHTMLAttributes<HTMLInputElement>,
+    'type' | 'onChange' | 'min' | 'max' | 'value' | 'id' | 'placeholder' | 'disabled'
+  > {
+  /** Additional class names */
+  className?: string | ClassValue;
+}
+/**
+ * Button component
+ *
+ * Example:
+ * ```js
+ * <Button className="text-black" loading disabled>Click Me</Typography>
+ * ```
+ */
+export const Input = ({ id, type, className, min, max, value, placeholder, disabled, onChange }: Props) => {
+  return (
+    <input
+      id={id}
+      role="input"
+      className={clsx('rounded border border-pink-800 p-2 outline-pink-300', className)}
+      type={type}
+      min={min}
+      max={max}
+      value={value}
+      placeholder={placeholder}
```

Comment:
**question** there are a lot of passed parameters here that don't differ by name, why didn't you use the rest operator?